apps/web/components/shared/qa-server.tsx

```
@@ -0,0 +1,136 @@
+  const { data: commentsData } = await supabase
+    .from("comments")
+    .select("*")
+    .eq("project_id", projectId)
+    .not("parent_comment_id", "is", null)
+    .order("created_at", { ascending: true });
+  
+  
+  let commentsWithAuthors: CommentData[] = [];
+  
+  if (commentsData && commentsData.length > 0) {
+    const authorIds = [...new Set(commentsData.map((item: any) => item.author_id))];
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)